compiler/damlc/tests/src/DamlcVisual.hs

```diff
@@ -26,4 +29,12 @@ unitTests = do
                 dotFile
                 path
                 (execVisual darPath (Just path))
+            , testCase "multiline manifest file test" $
```

Comment:
Maybe we should move it to a different module for Reader as in code, but as of now only visual module uses this so I put the test with the visual module

Reply:
It’s not quite true that only the visual module uses this. We also use the DAR reader in the test suite of the hs ledger bindings. But I’m fine with the test being here for now.

Reply:
Ah, I did not realize that it was used there as well. Thanks